packages/pulpcore/python-dynaconf/python-dynaconf.spec

```diff
@@ -2,8 +2,8 @@
 %global pypi_name dynaconf
 
 Name:           python-%{pypi_name}
-Version:        2.2.2
-Release:        2%{?dist}
+Version:        3.0.0rc1
```

Comment:
not sure we want to use an rc release of dynaconf, I'd revert the whole dynaconf change for now

Reply:
looking at https://github.com/evgeni/pulpcore-packaging/pull/3/files you explicitly want that version?

Reply:
Yes..This version contains a fix around auth for pulpcore that we rely on. We need the 3.0.0rc1 and that will go with pulpcore 3.2.1 which was released to be compatible with this.